mainnet/nethermind/nethermind.cfg

```diff
@@ -0,0 +1,27 @@
+{
+  "Init": {
+    "ChainSpecPath": "./configs/mainnet/shared/chainspec.json",
+    "GenesisHash": "0x5df88817dfb9b00d8ef142370671e8a9bc00c548ab78fbaf205df53db2b24a26",
+    "BaseDbPath": "./mainnet-data/execution"
+  },
+  "Sync": {
+    "SnapSync": true
+  },
+  "JsonRpc": {
+    "Host": "localhost",
+    "Port": 8551,
+    "Enabled": true,
+    "EnabledModules": ["eth","subscribe","trace","txpool","web3","personal","proof","net","parity","health","rpc","engine"],
```

Comment:
Please change Nethermind defaults to our custom setup: `eth, engine, web3, net`.

Reply:
Resolved